src/ResourceManager/ServiceBus/Commands.ServiceBus/Cmdlets/Namespace/GetAzureServiceBusNamespaceKey.cs

```diff
@@ -13,14 +13,17 @@
 // ----------------------------------------------------------------------------------
 
 using Microsoft.Azure.Management.ServiceBus.Models;
+using Microsoft.Azure.Commands.ServiceBus.Models;
 using System.Management.Automation;
+using System;
 
 namespace Microsoft.Azure.Commands.ServiceBus.Commands.Namespace
 {
     /// <summary>
     /// 'Get-AzureRmServiceBusNamespaceKey' Cmdlet gives key detials for the given ServiceBus Namespace Authorization Rule
     /// </summary>
-    [Cmdlet(VerbsCommon.Get, ServiceBusNamespaceKeyVerb), OutputType(typeof(ResourceListKeys))]
+    [ObsoleteAttribute("'Get-AzureRmServiceBusNamespaceKey' cmdlet is mark as obsolete and will be depricated in upcoming breaking changes build. Please use the New cmdlet 'Get-AzureRmServiceBusKey'", false)]
+    [Cmdlet(VerbsCommon.Get, ServiceBusNamespaceKeyVerb), OutputType(typeof(ListKeysAttributes))]
```

Comment:
@v-Ajnava scratch that, just saw you created this type from `ResourceListKeys` and now do the same thing for `AccessKeys` 😃